cishouseholds/pipeline/pipeline_stages.py

```diff
@@ -466,6 +467,7 @@ def execute_union_dependent_transformations(unioned_survey_table: str, transform
         output table name for table with applied transformations dependent on complete survey dataset
     """
     df = extract_from_table(unioned_survey_table)
+    df = reclassify_work_variables(df, spark_session=get_or_create_spark_session(), drop_original_variables=False)
```

Comment:
Could this instead be moved to line 777 after the assign_age_group_school_year function to ensure the age_at_visit variable is available.